packages/react/counter/src/Counter.tsx

```diff
@@ -118,18 +136,19 @@ export function Counter(props: CounterProps) {
     async function setup() {
       if (isStatic && value) {
         setDataValue(value)
-        return
       }
 
-      const fetchedValue = await fetchData()
+      if (!isStatic) {
+        const fetchedValue = await fetchData()
 
-      if (!fetchedValue) {
-        setHasError(true)
-        console.error(`QueryError: Your metric ${query?.metric} returned undefined or a \`null\` value.`)
-        return
-      }
+        if (typeof fetchedValue === 'undefined') {
+          setHasError(true)
+          console.error(`QueryError: Your metric ${query?.metric} returned undefined.`)
```

Comment:
@felipecadavid I am think we should probably remove the `console.error` for now, and plan for logging as a feature soon.

What I mean is that,

1. First of all, customers should be able to enable/disable logging; and
2. Second of all, customers may even want to provide their own callbacks to avoid sending directly to console.

Since we don't have that prepared yet, let's not add more non-configurable `console.error`, `console.log`, etc., calls to the library just yet.
```suggestion
```

Reply:
I'm going to comment all the `console.error` and will leave a comment related to this